fasthtml/components.py

```diff
@@ -19,7 +19,8 @@
 # %% ../nbs/api/01_components.ipynb
 from dataclasses import dataclass, asdict, is_dataclass, make_dataclass, replace, astuple, MISSING
 from bs4 import BeautifulSoup, Comment
-from typing import Literal, Optional
+from collections import UserDict
```

Comment:
Looks like this is only used for a test/example, so it shouldn't be exported. To fix this, move this import into the cell that does not have a `#| export` header.

Reply:
Aha. I thought that was weird that the test import went into the actual source. Didn't see how that was working. Fixed.